doc/api/path.md

```diff
@@ -68,7 +68,8 @@ changes:
 * Returns: {string}
 
 The `path.basename()` methods returns the last portion of a `path`, similar to
-the Unix `basename` command.
+the Unix `basename` command. Trailing `/` (and `\\` on Windows) characters are
+ignored.
```

Comment:
This is preexisting (I copied that paragraph from `path.md` about `path.win32`). Is it to be considered a bug in `dirname` or is it a documentation inaccuracy?